### ISSUE TITLE
[BUG] Can't change user roles without giving them Superadmin role

### ISSUE THREAD

**Issue by delford8** [MakerLab-Dev/MakerLab-Web#2]
**Describe the bug**
Using my Superadmin account I set a Standard user roles to a new role I created and removed the "Standar user" role but it changed to the new role I created and to Superadmin 

**To Reproduce**
1. Create a user
2. With a Superadmin account (or an admin account) change the user roles from "Standar user" to another role, admin for example
3. When you click save, you will see that the user now has a Superadmin role too
4. Note: Admins can exploit this to get Superadmin

**Expected behavior**
It shouldn't set the Superadmin role if it wasn't on the list

**Server(please complete the following information):**
 - OS: Windows 10
 - Server: Apache/2.4.38
 - PHP Version: 7.3.2
 - MySQL Version: 10.1.38-MariaDB 
 - Version: 0.5 from beta branch (commit b65b86860d18392b6a7650014b44df34aaa0e5d8)

**Desktop (please complete the following information, If applicable):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 73.0.3683.103

**Comment by MakerLab-Dev** [comment]
Thank you for your contribution!😊
The bug should be fixed now, if you have a suggestion, found another bug or want to contribute in other ways, you can open another issue, contribute code or contribute to the Wiki!
This issue will close once commit 5f9e312 is merged into the 'master' branch
